--- C:/wip.home/mcp41/src.new/minecraft/net/minecraft/src/GuiConnectFailed.java	Mon Jun 13 10:38:29 2011
+++ C:/wip.home/mcp41/src/minecraft/net/minecraft/src/GuiConnectFailed.java	Mon Jun 13 11:37:35 2011
@@ -24,7 +24,6 @@
         {
             errorDetail = stringtranslate.translateKey(s1);
         }
-        Transporter.detectKick(s, s1, aobj);
     }
 
     public void updateScreen()
@@ -56,7 +55,6 @@
         drawCenteredString(fontRenderer, errorMessage, width / 2, height / 2 - 50, 0xffffff);
         drawCenteredString(fontRenderer, errorDetail, width / 2, height / 2 - 10, 0xffffff);
         super.drawScreen(i, j, f);
-        Transporter.teleportIfKicked(mc);
     }
 
     private String errorMessage;
